api/docs/eigenda-protos.md

```diff
@@ -0,0 +1,1661 @@
+# Protocol Documentation
+<a name="top"></a>
+
+## Table of Contents
+
+- [churner/churner.proto](#churner_churner-proto)
```

Comment:
It's unclear if it's best to put all docs together. 
There 3 categories of APIs for different roles: operators, users (consumers), and admin (-ish, like churner/ejector). It may be clearer for people to visit a page with APIs relevant to them.

Reply:
The script now generates both the individual documents for each proto file as well as a unified document in case somebody prefers that format. If you don't think it's helpful, I can delete the unified file.